viadot/flows/sap_rfc_to_adls.py

```diff
@@ -4,48 +4,34 @@
 
 from viadot.tasks import SAPRFCToDF
 from viadot.tasks import AzureDataLakeUpload
-from viadot.task_utils import df_to_csv
+from viadot.task_utils import df_to_csv, df_to_parquet, concat_dfs
 
 download_sap_task = SAPRFCToDF()
 file_to_adls_task = AzureDataLakeUpload()
 
 
-@task
-def concat_dfs(dfs: List[pd.DataFrame]):
-    """
-    Task to combine list of data frames into one
-
-    Args:
-        dfs (List[pd.DataFrame]): List of dataframes to concat.
-    Returns:
-        full_df (pd.DataFrame()): Pandas dataframe containing all columns from dataframes from list.
-    """
-    full_df = pd.DataFrame()
-    for df in dfs:
-        full_df = pd.concat([full_df, df], axis=1)
-    return full_df
-
-
 class SAPRFCToADLS(Flow):
     def __init__(
         self,
         name: str,
-        query_list: List[str],
-        sep: str = None,
+        query: str = None,
+        queries: List[str] = None,
+        rfc_sep: str = None,
         func: str = "BBP_RFC_READ_TABLE",
```

Comment:
default `func` should be consistent with the default in the source (so either set to the same or change in both places if appropriate

Reply:
same as above :)